### PR TITLE
Echo query parameters

### PR DESCRIPTION
**Pull request by admiral96**
Extracting parameter given from EngageSPARK Webhook engagement

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -10,7 +10,9 @@ def index():
 @url.route('/query')
 def show_user_input():
     content = request.args.get('content')
-    return content or ("You've entered an incorrect/empty query. Please check and try again. You can try '/query?content=something relevant'", 400)
+    sms_from = request.args.get('from')
+    sms_to = request.args.get('to')
+    return 'Content: ' + content + ' SMS From: ' + sms_from + '  SMS To: ' + sms_to or ("You've entered an incorrect/empty query. Please check and try again. You can try '/query?content=something relevant'", 400)
```
Comment (mknecht) [inline app/views.py]: can you correct your error text?

Also, the `or` operator is not appropriate anymore, as the first operand will _never_ be falsy

**Comment by mknecht** [comment]
can you add two tests for from or two number being empty?

**Comment by admiral96** [comment]
on a different PR? @mknecht

**Comment by mknecht** [comment]
@admiral96 nope: in this PR you introduce a new feature (new parameters), and of course the tests come with the feature.

**Comment by admiral96** [comment]
got it

**Comment by mknecht** [comment]
lgtm, but the tests seem a bit cryptic.

can you review with marisa? particularly, show her your tests. if she does not understand something, figure out with her how you can improve it. thanks
